### PR TITLE
Use MyST admonition syntax

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: docs/source/configuration/settings-reference.md [hunk 1]
```diff
@@ -10,27 +10,23 @@ html_meta:
...
+```{note}
+This list is still incomplete, contributions are welcomed!
+```
 
-### navDepth
+## Main settings
 
-!!! block ""
+```{glossary}
+:sorted: true
```
Comment (stevepiercy) [inline docs/source/configuration/settings-reference.md]: Clever!

I removed `true` as it is optional.

**Review by stevepiercy — APPROVED** [review]
This looks good.

I fixed up the indentation in one file and added `html_meta` tags for two files I touched. @sneridagh please take a look, and let me know.

**Comment by sneridagh** [comment]
@stevepiercy looks good too!

**Comment by stevepiercy** [comment]
I had to fix some merge conflicts, and force pushed. I'm going to wait only on the docs build task to complete and pass, then merge.

**Comment by stevepiercy** [comment]
Docs build passed. Merging!

**Pull request merged**